Library/Formula/mono.rb

```diff
@@ -2,8 +2,8 @@
 
 class Mono < Formula
   homepage "http://www.mono-project.com/"
-  url "http://download.mono-project.com/sources/mono/mono-3.10.0.tar.bz2"
-  sha1 "74e43604ea48e941c39a43ebc153abee4ddba56c"
+  url "http://download.mono-project.com/sources/mono/mono-3.12.0.tar.bz2"
+  sha256 "8fdc69329c8d669d1c5e5bc922a5df8a3ed61a72344b92e5a073809bf1b36783"
```

Comment:
Just leave this as `sha1` unless it's e.g. prominently displayed as `sha256` on the homepage.